code/modules/mob/living/carbon/alien/alien.dm

```diff
@@ -27,7 +30,7 @@
 	verbs += /mob/living/proc/ventcrawl
 	verbs += /mob/living/proc/hide
 
-	name = "[initial(name)] ([rand(1, 1000)])"
+	name = "([get_descriptor()] [initial(name)]"

```

Comment:
Simple mobs, especially ghost controlled mobs should not be named in this way, its generally a un-needed bloaty change.

Reply:
Why is it not needed. Can you explain why it is bloaty? This is a fundamental component of the PR, and you have not provided any context as to why it needs to be changed other than 'I think it's unncessary'